### PR TITLE
Feature/include your own tweets

### PR DESCRIPTION
**Pull request by hjertnes**
- Add config option to include your own tweets in the timeline

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /build
 /twet
+.idea
```
Comment (quite) [inline .gitignore]: Pls dont do this. I suggest to keep editor specific ignores and such in a global git excludesfiles in your setup.

File: config.go [hunk 2]
```diff
@@ -27,6 +27,7 @@ type Config struct {
 	DiscloseIdentity bool
 	Timeline         string
 	Hooks            Hooks
+	IncludeYourself  bool
```
Comment (quite) [inline config.go:30]: Please also add a commented out instance of this option with its defualt value to config.yaml.example (with a comment line explaining what it does)

**Review by quite — COMMENTED** [review]
Nice one! Just some minor adjustments and docs requested

File: .gitignore (hunk 1 above)
Comment (hjertnes) [reply .gitignore]: Sure :)

File: config.go (hunk 2 above)
Comment (hjertnes) [reply config.go:30]: 👍

**Comment by hjertnes** [comment]
I've done the changes you requested 😄

File: config.yaml.example [hunk 3]
```diff
@@ -9,6 +9,9 @@ twtfile: ~/public_html/twtxt.txt
 # add User-Agent header with nick and program version when fetching feed (default true)
 #discloseidentity: true
 
+# include your own posts in the timeline
+# includeyourself: true
```
Comment (quite) [inline config.yaml.example]: I agree that this should be the default. The default value of a bool in Go is false though. Maybe you should add a true default to the config struct in main.

Comment (quite) [reply config.yaml.example]: I mixed up of course! I think the default should be false, thus not changing the current behaviour. So putting something in the config struct wont be needed. Just change this to false.

Comment (hjertnes) [reply config.yaml.example]: Done

**Review by quite — APPROVED** [review]

**Pull request merged**